<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggerWindow</class>
 <widget class="QMainWindow" name="LoggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="btnStart">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/resources/images/play.png</normaloff>:/resources/images/play.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>17</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnStop">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/resources/images/stop.png</normaloff>:/resources/images/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>17</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnDump">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Download Log</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/resources/images/plot.png</normaloff>:/resources/images/plot.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>17</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>Save Log</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>view/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
